.gitattributes

```diff
@@ -22,3 +22,6 @@
 *.woff binary
 *.pyc binary
 *.swp binary
+
+# Libs
+*.aar binary
```

Comment:
This is unneeded 03235e15e386e332ff753c5edfc55a3ffa368a7a

Reply:
aaaaaa